### PR TITLE
Task 9 first review

### PR DESCRIPTION
**Pull request by Maksym-Lizohub**

### CODE REVIEW

File: app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/image_with_borders.phtml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/image_with_borders.phtml]: Remove Magento copyrights please, and add template annotation about changes

File: app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/list.phtml [hunk 2]
```diff
@@ -0,0 +1,177 @@
+<?php
+/**
+ * 
+ */
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/list.phtml:9]: Add template annotation about changes

File: app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/list/toolbar/amount.phtml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/list/toolbar/amount.phtml]: Remove Magento copyrights please, and add template annotation about changes

File: app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/list/toolbar/sorter.phtml [hunk 4]
```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/list/toolbar/sorter.phtml]: Remove Magento copyrights please, and add template annotation about changes

File: app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/list/toolbar/viewmode.phtml [hunk 5]
```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+?>
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/list/toolbar/viewmode.phtml]: Remove Magento copyrights please, and add template annotation about changes

File: app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_catalog-page.less [hunk 6]
```diff
@@ -1,8 +1,270 @@
-.columns {
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_catalog-page.less:1]: Please add `& when (@media-common =true) {}` wrapper

File: app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_catalog-page.less [hunk 7]
```diff
@@ -1,8 +1,270 @@
...
+        }
+        .limiter-options {
+            height: 36px;
+            margin: 0;
+            font-size: 13px;
+            color: #888888;
+            border-color: #eee;
+        }
     }
 }
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_catalog-page.less:272]: Please check this file and add variables and libs where appropriate

**Review by y-mantuliak — CHANGES_REQUESTED** [review]
also please format all your custom less files like _homepage.less (spacing etc)
[app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less](https://github.com/Maksym-Lizohub/magento/pull/8/files#diff-d85c7c14e6d21ac40b3ff367ad06eef2eb2ba547dfea7a3871db23dfe5916098)

**Comment by y-mantuliak** [comment]
Also format all your custom less files like _homepage.less (spacing etc)
[app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less](https://github.com/Maksym-Lizohub/magento/pull/8/files#diff-d85c7c14e6d21ac40b3ff367ad06eef2eb2ba547dfea7a3871db23dfe5916098)

File: app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/image_with_borders.phtml (hunk 1 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/image_with_borders.phtml]: File for test. Deleted

File: app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/list.phtml (hunk 2 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/list.phtml:9]: Add code for image for hover. Add anotation.

Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/list.phtml:9]: done

File: app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/list/toolbar/amount.phtml (hunk 3 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/list/toolbar/amount.phtml]: Copied folder "toolbar" with all files. Files amount deleted.

File: app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/list/toolbar/sorter.phtml (hunk 4 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/list/toolbar/sorter.phtml]: Copied folder "toolbar" with all files. Files sorter deleted.

File: app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/list/toolbar/viewmode.phtml (hunk 5 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/list/toolbar/viewmode.phtml]: Copied folder "toolbar" with all files. File  viewmode deleted.

File: app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_catalog-page.less (hunk 6 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_catalog-page.less:1]: Done

File: app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_catalog-page.less (hunk 7 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_catalog-page.less:272]: Done

**Review by y-mantuliak — APPROVED** [review]

**Pull request merged**